Dockerfile

```diff
@@ -17,17 +17,17 @@
 # Now you can connect to:
 #    http://172.17.0.1:8891
 #
-FROM ubuntu:14.04
+FROM ubuntu:16.04
 MAINTAINER pietro.delsante_AT_gmail.com
 ENV DEBIAN_FRONTEND noninteractive
 EXPOSE 8891
 
 RUN apt-get update && \
-    apt-get -y install curl && \
+    apt-get -y install curl sudo && \
     curl -sL https://deb.nodesource.com/setup_4.x | sudo bash - && \
     apt-get -y install nodejs build-essential xvfb libfontconfig1 && \
-    npm install -g slimerjs@0.9.6-2 && \
-    npm install -g phantomjs@1.9.19 && \
-    npm install -g manet@0.4.10
+    npm install -g slimerjs@0.906.2 && \
```

Comment:
It looks like the latest official version of slimerjs is 0.10: https://slimerjs.org/download.html

But maybe we can simply drop the version numbers and let npm download the latest version on its own?